libs/monitor.js

```diff
@@ -55,6 +56,12 @@ function initRoutes(cb) {
 
   //app routes
   app.get(/\/$|\/admin$/, require('../server/routes/index'));
+  app.use("/box/srv/1.1/admin/authpolicy/auth", require("../server/routes/auth"));
```

Comment:
With the way the fh-js-sdk works right now this was the least intrusive way to do it I could come up with. The short of it is, the url needs the fh_host query parameter for $fh.init to work which puts it into local mode which makes the auth point to the service itself instead of millicore. So I put a proxy in the service. Otherwise it won't work when running in the current iteration of the FeedHenry Studio service preview.